docs/clients/arming_handler.rst

```diff
@@ -0,0 +1,149 @@
+Arming Handler
+--------------
+Vertiq Advanced Speed modules can support an advanced arming feature, 
```

Comment:
Can you do a link to the advanced arming section in the Feature Reference Manual section here, so people can see that documentation easily?